contracts/polydex/Migrator/PolydexMigrator.sol

```diff
@@ -130,7 +130,7 @@ contract PolyDexMigrator {
                 keccak256(abi.encodePacked(token0, token1)),
                 // Init code hash. It would be specific each exchange(different for quickswap, dfyn, etc).
                 // So when deploying Migrator, change it.
-                hex'eb126fcec23a301a6ca69c92882e216c77c0fb35a8bb4a45e82ac47d18d0cc3e'
+                hex'8cb41b27c88f8934c0773207afb757d84c4baa607990ad4a30505e42438d999a'
```

Comment:
This Init code hash should of the oldRouter( the one which we are migrating from in this case 
if Quickswap: 96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f & 
taken from https://polygonscan.com/address/0xa5e0829caced8ffdd4de3c43696c57f7d7a678ff#code

if Sushiswap : e18a34eb0e04b04f7a0ac29a6e80748dca96319b42c54d679cb821dca90c6303 
reference: https://polygonscan.com/address/0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506#code

Also make 2 different migrator files one for quickswap & other for sushiswap with there respective init hash